saleor/plugins/webhook/tasks.py

```diff
@@ -270,8 +271,9 @@ def trigger_all_webhooks_sync(
         if webhook.subscription_query:
             if request_context is None:
                 request_context = initialize_request(
-                    requestor, event_type in WebhookEventSyncType.ALL
+                    requestor, event_type in WebhookEventSyncType.ALL, not allow_replica
```

Comment:
When we change is_mutation to allow_replica this should become: 
```suggestion
                    requestor, event_type in WebhookEventSyncType.ALL, allow_replica
```